### ISSUE TITLE
Not able to get it working

### ISSUE THREAD

**Issue by emekaelo** [martingrosche/notion-2-issue#10]
## Describe the bug
The issues don't show up in the GitHub repo after it's been created and then manually triggering the action

##  To Reproduce
List of steps to reproduce the behavior.
I followed the steps in the action marketplace

## Expected behavior
I expected the issues created in Notion to appear on the GitHub repo

## Versions
- Action: 1.0.0
- Notion: 2.36

## Additional context
It's a private repo

Workflow below
```yml
name: Notion Database
on: workflow_dispatch
permissions:
  contents: read
  repository-projects: read
  issues: write
jobs:
  sync-github-issues:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v2
      - name: Run Action
        uses: martingrosche/notion-2-issue@v1.0.0
        with:
          notionToken: ${{ secrets.NOTION_TOKEN }}
          notionDatabase: ${{ secrets.NOTION_DATABASE }}

```

**Comment by martingrosche** [comment]
Hello 

Could you please be more specific? Do you got any error message within the action?

I am using 2.36 notion version as well. How can I reproduce the issue?

**Comment by emekaelo** [comment]
The action runs successfully
No error within the action. However, there's a warning
![image](https://github.com/martingrosche/notion-2-issue/assets/25552096/4211802c-e273-4633-b027-b203f09ca1a6)

**Comment by emekaelo** [comment]
There's also this warning
![image](https://github.com/martingrosche/notion-2-issue/assets/25552096/bb47f2d3-d534-4aaf-92dd-563ee6334e7e)

**Comment by martingrosche** [comment]
Ok there are two parts
- Warning `"Title" property must be not empty` -> please take a look at the [notion database template](https://github.com/martingrosche/notion-2-issue/tree/main#notion-database-template). There has to be a table header called `Title`
- using `actions/checkout@v2` -> you are using the v2 in your GitHub action. Please see [GitHub checkout action](https://github.com/marketplace/actions/checkout) for more details. This is not part of my action itself. 

Hope I could help you.

**Comment by emekaelo** [comment]
Yes! It works!
The first one helped.

When reading the guide, I had created a new table in notion for it as I was following the steps. That table did not have a column with `Title` header. Then I also duplicated the template. However, I used the database ID of the first table. So I deleted the first table and updated the value for `NOTION_DATABASE` secret with the database Id of the second table(template).

Thank you so much.
